views/includes/check-your-answers/beneficial-owner-individual.html

```diff
@@ -74,7 +74,7 @@ <h3 class="govuk-heading-m">Individual beneficial owner</h3>
       actions: {
         items: [ CREATE_CHANGE_LINK(
           OE_CONFIGS.BENEFICIAL_OWNER_INDIVIDUAL_URL + "/" + boi.id + OE_CONFIGS.LAST_NAME,
-          "Individual beneficial owner " + boi.first_name + " - last name",
+          "Individual beneficial owner " + boi.first_name + " " + boi.last_name + " - last name",
```

Comment:
Just reading the defect and actually wondering if the issue here was simply that this line originally referenced the boi.first_name field and not the boi.last_name field? Then it would inform user of the correct data... rather than a fix that includes both fields?

Reply:
The original goal of this was that It's meant to be so the user can see the full name of the BO and what it's changing, so "Jim Jenson - last name". Same for both the fields

Reply:
Ah right. I think I now follow.

Another question though - Chrissie has highlighted in yellow the 'date of birth' and 'nationality' fields also as only displaying first name. Should these be fixed too?

Reply:
Potentially, I'll ask now

Reply:
See you've change the code - looks good. Did Chrissie say yes to all links needing this then? If yes we should get the story description updated to make this clear to testers too.

Reply:
Yes she did, I'll ask her to update the story 👍

Reply:
Updated

Reply:
👍 thanks.